WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -1,4 +1,8 @@
-﻿/* Variation ID. Parameters: %1$@ - Product variation ID */
+/* Generated File. Do not edit. */
+
+/* MARK: - Localizable.strings */
+
+/* Variation ID. Parameters: %1$@ - Product variation ID */
```

Comment:
The `/* Generate File...` as the first line shows these changes were brought about by the new localization tooling 👍 

This means we (hopefully) won't run into the issue experienced during the 9.3 release finalization 👌